numpy/polynomial/laguerre.py

```diff
@@ -1441,10 +1441,11 @@ def lagfit(x, y, deg, rcond=None, full=False, w=None):
     --------
     >>> from numpy.polynomial.laguerre import lagfit, lagval
     >>> x = np.linspace(0, 10)
-    >>> err = np.random.randn(len(x))/10
+    >>> rng = np.random.default_rng()
+    >>> err = rng.normal(scale=1./10,size=len(x))
```

Comment:
```suggestion
    >>> err = rng.normal(scale=1./10, size=len(x))
```

Reply:
Thank you for the white space reminder and for letting me fix it rather than just changing it and merging it in. The training is helpful.